lib/drab/commander.ex

```diff
@@ -240,6 +262,92 @@ defmodule Drab.Commander do
   Drab injects function `render_to_string/2` into your Commander. It is a shorthand for
   `Phoenix.View.render_to_string/3` - Drab automatically chooses the current View.
 
+  ## PubSub support
```

Comment:
Technically Drab already has PubSub support as of https://github.com/grych/drab/releases/tag/v0.8.3 although not of the form you are expecting, rather it passes messages to the javascript to then be handled there.  Messages to the Drab process should already be web-ready regardless, unknown pubsub messages should never reach drab regardless.

Reply:
It seems that Drab hasn't any kind of PubSub support. It just use PubSub for its internal uses, that it is a different things to let the developer uses Phoenix.PubSub directly inside a live page if needed, as for example LiveView does and suggest.